src/main/java/com/neuvector/model/NVScanner.java

```diff
@@ -18,12 +19,13 @@ public NVScanner(){}
      * @param nvRegistryUser  The user name to login the registry. If the user name and the password are empty, the API will skip the docker login action.
      * @param nvRegistryPassword The password to login the registry.
      */
-    public NVScanner(String nvScannerImage, String nvRegistryURL, String nvRegistryUser, String nvRegistryPassword, Logger log){
+    public NVScanner(String nvScannerImage, String nvRegistryURL, String nvRegistryUser, String nvRegistryPassword, Logger log, Boolean bindMountShared){
```

Comment:
Javadoc for the method (also for the one on line 37) needs an update.

Reply:
Commit: [800043c](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/800043c6090af4b71dc352175d4d412c5917267b)